crates/ruff_python_formatter/src/expression/expr_named_expr.rs

```diff
@@ -33,14 +32,11 @@ impl FormatNodeRule<ExprNamedExpr> for FormatExprNamedExpr {
 impl NeedsParentheses for ExprNamedExpr {
     fn needs_parentheses(
         &self,
-        parenthesize: Parenthesize,
-        context: &PyFormatContext,
-    ) -> Parentheses {
-        match default_expression_needs_parentheses(self.into(), parenthesize, context) {
-            // Unlike tuples, named expression parentheses are not part of the range even when
-            // mandatory. See [PEP 572](https://peps.python.org/pep-0572/) for details.
-            Parentheses::Optional => Parentheses::Always,
-            parentheses => parentheses,
-        }
+        _parent: AnyNodeRef,
+        _context: &PyFormatContext,
+    ) -> OptionalParentheses {
+        // Unlike tuples, named expression parentheses are not part of the range even when
+        // mandatory. See [PEP 572](https://peps.python.org/pep-0572/) for details.
```

Comment:
it's actually a TODO on my end
```suggestion
        // mandatory. TODO(konstin): Implement [PEP 572](https://peps.python.org/pep-0572/) rules.
```